src/Umbraco.Web/Redirects/RedirectUrlSearchResults.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using Umbraco.Core.Models;
+
+namespace Umbraco.Web.Redirects
+{
+    public class RedirectUrlSearchResult
+    {
+        public string StatusMessage { get; set; }
```

Comment:
Turns out `StatusMessage ` is not used at all and the request to get the status shouldn't be part of the search result (i.e. `UrlTrackerDisabled `)